pypesto/problem.py

```diff
@@ -118,6 +130,12 @@ def __init__(self,
             x_names = [f'x{j}' for j in range(0, self.dim_full)]
         self.x_names: List[str] = [name for name in x_names]
 
+        if x_scales is None:
+            x_scales = ['lin'] * self.dim_full
+        self.x_scales = x_scales
+
+        self.x_priors = x_priors_defs
```

Comment:
so, currently the priors are just added to the problem, but not used in any form for optimization or sampling yet, right?